toolchains/toolchain.bzl

```diff
@@ -0,0 +1,154 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under both the MIT license found in the
+# LICENSE-MIT file in the root directory of this source tree and the Apache
+# License, Version 2.0 found in the LICENSE-APACHE file in the root directory
+# of this source tree.
```

Comment:
I've just put up D42040624 to move the toolchains into the prelude, so they can be reused here rather than copied (one of them has broken in the last two weeks since you released this)